src/andamio/course/100/1001.md

```diff
@@ -6,15 +6,16 @@
   "description": "Install a Cardano Browser Wallet",
   "videoURL": "https://www.youtube.com/watch?v=4YQsIQpAQ8E",
   "lastEdited": "2023-12-09",
-  "author": "Z"
+  "author": "Z",
 }
 ---
 
-Most users interact with dApps via ***browser wallets***, which usually come in the form of extensions to their browser to enable them to interact easily with the blockchain.  Mesh has support for Cardano wallets which accord with [CIP-30 ("Cardano Improvement Proposal 30")](https://cips.cardano.org/cip/CIP-0030), and a pre-requisite for this course is that you have at least one installed.
+Most users interact with dApps via **_browser wallets_**, which usually come in the form of extensions to their browser to enable them to interact easily with the blockchain. Mesh has support for Cardano wallets which accord with [CIP-30 ("Cardano Improvement Proposal 30")](https://cips.cardano.org/cip/CIP-0030), and a pre-requisite for this course is that you have at least one installed.
 
-There are a number of different wallets to choose from, including [Eternl](https://eternl.io/), [Lace](https://www.lace.io/), [Nami](https://namiwallet.io/), [Typhon](https://typhonwallet.io/) and [others](https://docs.cardano.org/new-to-cardano/types-of-wallets/).  While they all vary in terms of their design, workflow and some details, so long as the implementation faithfully complies with CIP-30, they should offer a common API enabling the dApps to communicate with them.
```

Comment:
What wallet names to show / order of it might signal something. Not sure if we want to come back to this before release

Reply:
Let me create an issue, let's check again pre-launch.

Reply:
@jinglescode can you help enable the issue function in this repo, to keep track on todos?